src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -19,7 +19,7 @@ public final class Constants {
 
     public static final int PASSWORD_MIN_LENGTH = 8;
 
-    public static final int PASSWORD_MAX_LENGTH = 50;
+    public static final int PASSWORD_MAX_LENGTH = 100;
```

Comment:
Was this intentionally changed in this PR?

Reply:
I would assume it was, but I'm not sure tbh @Hialus